### DISCUSSION TITLE
Question on using .crs - Rioxarray Help

### DISCUSSION THREAD

**Discussion by minananada** [giswqs/geog-312#113] [Q&A] [answered]
Hi all,

I'm following the labs through the YouTube playlist and I'm struggling to understand why the output of  `data.rio.crs`  is:

`CRS.from_wkt('PROJCS["WGS 84 / UTM zone 11N",GEOGCS["WGS 84",DATUM["World Geodetic System 1984",SPHEROID["WGS 84",6378137,298.257223563]],PRIMEM["Greenwich",0],UNIT["degree",0.0174532925199433],AUTHORITY["EPSG","4326"]],PROJECTION["Transverse_Mercator"],PARAMETER["latitude_of_origin",0],PARAMETER["central_meridian",-117],PARAMETER["scale_factor",0.9996],PARAMETER["false_easting",500000],PARAMETER["false_northing",0],UNIT["metre",1],AXIS["Easting",EAST],AXIS["Northing",NORTH],AUTHORITY["EPSG","32611"]]')`

I'm expecting to just get something like: `EPSG:32611`

Why is printing out the WKT (Well-known text) and not the shorthand EPSG code? Any help here would be greatly appreciated!

![Screenshot 2024-10-14 104000](https://github.com/user-attachments/assets/d15b91e6-c4d1-48d5-b30b-206102048c44)

**Comment by giswqs** [comment, marked as answer]
The output you’re seeing from data.rio.crs is the complete Well-Known Text (WKT) representation of the Coordinate Reference System (CRS), which includes detailed information about the projection, datum, and units.

If you're expecting a simpler output like EPSG:32611, you can use the to_epsg() method to extract the EPSG code directly. Here’s how you can do it:

```python
epsg_code = data.rio.crs.to_epsg()
print(epsg_code)
```

**Reply by ghost** [reply]
see #121  . Why is the return value of epsg_code None?

**Reply by minananada** [reply]
I sometimes get this result too
